webapp/help_post.py

```diff
@@ -46,6 +49,41 @@ def send_help_email(username:str, user_email:str, subject:str, message:str) -> b
         return False
     return True
 
+def send_sudo_request_email(username:str, user_email:str):
+    """
+    Sends an email notifying SysAdmins that a user has requested an account on feynman.
+
+    :param username the server username of the user who made the request.
+    :param user_email the email address of that user to contact them for vetting.
+    """
+    message_body = \
+    """
+Hi {username},
+
+Thank you for making a request for an account with sudo privileges on feynman.netsoc.co.
+
+We will be in touch shortly. 
+
+Best,
+
+The UCC Netsoc SysAdmin Team.
+
+PS: Please "Reply All" to the emails so that you get a quicker response.
+
+""".format(username=username)
+    
+    msg = EmailMessage()
+    msg.set_content(message_body)
+    msg["From"] = p.NETSOC_ADMIN_EMAIL_ADDRESS
+    msg["To"] = "adamgillessen@gmail.com"# p.NETSOC_EMAIL_ADDRESS
```

Comment:
Is this correct or was this just for testing? :P

Reply:
That's for testing. Nice catch :P